### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/layout_product-cards/
https://panianchuk.github.io/layout_product-cards/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,60 @@
...
+        >
+      </div>
+
+      <div class="card__container-body">
+        <h2 class="card__product-name">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <div class="card__product-code">
+          Product code: 195434
+        </div>
```
Comment (SanyaBratashchuk) [inline src/index.html:29]: image

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,136 @@
+* {
+  padding: 0;
+  margin: 0;
```
Comment (SanyaBratashchuk) [inline src/style.css:3]: Do not appply any styles to the `*` selector except `box-sizing`

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+.stars {
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
```
Comment (SanyaBratashchuk) [inline src/style.css:72]:
```suggestion
.stars__star {
```

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
```
Comment (SanyaBratashchuk) [inline src/style.css:66]: move each BEM block to the seperate file

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,60 @@
...
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <div class="reviews">
```
Comment (SanyaBratashchuk) [inline src/index.html:38]:
```suggestion
          <div class="card__reviews">
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
add demo & test report link to your pull request

**Comment by panianchuk** [comment]
https://panianchuk.github.io/layout_product-cards/ 
https://panianchuk.github.io/layout_product-cards/report/html_report/

File: src/index.html [hunk 6]
```diff
@@ -1,13 +1,60 @@
...
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__container-image">
```
Comment (alexandra-protyanova) [inline src/index.html:15]: redundant div

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,60 @@
...
+        <img 
+          src="./images/imac.jpeg" 
+          alt="imac" 
+          class="card__image"
+        >
+      </div>
+
+      <div class="card__container-body">
```
Comment (alexandra-protyanova) [inline src/index.html:23]: redundant div as well

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+.card {
+  display: flex;
+  flex-direction: column;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  max-width: 200px;
```
Comment (alexandra-protyanova) [inline src/style.css:22]: do not set width directly to bem blocks, use bem mixes instead

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+  color: #fff;
+}
+
+.card__button:hover {
+  color: #00acdc;
+  background: #fff;
+  border: 1px solid #00acdc;
+}
+
+.stars {
```
Comment (alexandra-protyanova) [inline src/style.css:114]: move each bem block to separate file

File: src/style.css [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+  display: flex;
+}
+
+.stars--0 .stars__star,
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
```
Comment (alexandra-protyanova) [inline src/style.css:123]: why just not to use `.stars__star` as selector, nothing will change, but it will be less code

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

**Pull request closed without merging**